CONTRIBUTION.md

```diff
@@ -202,7 +202,10 @@ $ git push origin feature/your-feature-name
 ```
 
 ### Request Pull
-Request a pull from "shader-slang" repository.
+
+Open a pull request against `shader-slang/slang`.
+
+Code formatting can be automatically fixed on your branch by commenting `/format`, a bot will proceed to open a PR targeting *your* branch.
```

Comment:
The formatting is a required process not an optional step. I think we should describe that the generated PR needs to be merged here as Yong pointed out on the other spot.